src/components/Signup/PodRegistrationForm.jsx

```diff
@@ -0,0 +1,152 @@
+// React Imports
+import React, { useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+
+// MUI imports
+import TextField from '@mui/material/TextField';
+import Typography from '@mui/material/Typography';
+import { Visibility, VisibilityOff } from '@mui/icons-material';
+import FilledInput from '@mui/material/FilledInput';
+import IconButton from '@mui/material/IconButton';
+import InputAdornment from '@mui/material/InputAdornment';
+import InputLabel from '@mui/material/InputLabel';
+import Button from '@mui/material/Button';
+import Card from '@mui/material/Card';
+import CardHeader from '@mui/material/CardHeader';
+
+/* Styles for MUI components */
+const cardStyle = {
+  padding: '8px',
+  margin: '8px',
+  boxShadow: '0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19)'
```

Comment:
I don't think a final design for signup has been decided yet. But right now it looks like two components layered on top of each other, which to me is a bit strange. `Set Up a New Pod` right on top of `Register for PASS` seems like too much. Maybe others will disagree, but I think they can be combined to look more like a single component. This would mean removing the border and box shadow for `Set Up a New Pod`.

Reply:
@andycwilliams I briefly spoke to @leekahung about this and we sort of thought restyling would be out of scope of this ticket. I simply took what was being used on the Signup page and broke into small digestible code.

Reply:
Yeah, think we could keep this PR dedicated to refactoring. We could get the styling updated in a subsequent/follow-up PR.

Reply:
@andycwilliams @leekahung I started making a few changes based on feedback and things definitely started to shift around. IMHO we should maybe loop back on the style changes in a new task. Is that cool with you guys?

Reply:
Since I'm away at the moment (didn't bring my computer with me on trip) I could mainly just do comments or replies. Think of the others are okay with it, I'm good.

Reply:
I'm fine with changing look/styling in another PR. Maybe go ahead and make that issue, but say that this issue needs to be resolved before starting work on that one - just so nobody picks it up and creates merge conflicts?